locale/en/download/package-manager.md

```diff
@@ -72,10 +73,20 @@ dnf module install nodejs:12
 
 For CentOS/RHEL 7 Node.js is available via [Software Collections](https://www.softwarecollections.org/en/scls/?search=NodeJS).
 
-## Debian and Ubuntu based Linux distributions, Enterprise Linux/Fedora and Snap packages
+### Alternatives
+
+These resources provide packages compatible with CentOS, Fedora, and RHEL.
```

Comment:
I think this should be "These resources provide packages compatible with CentOS, Fedora, and RHEL. if you don't need support"

Reply:
or something along those lines so it's clear that you won't get support if you use those versus the ones in software collections.

Reply:
Isn't that information covered by the paragraph at the top of the page?

> ***Note:*** The packages on this page are maintained and supported by their respective packagers, **not** the Node.js core team. Please report any issues you encounter to the package maintainer. If it turns out your issue is a bug in Node.js itself, the maintainer will report the issue upstream.

Reply:
What I'm trying to get across is that Red Hat only supports the packages which are installed via software collections.  So while the alternatives are worth mentioning, \you won't get support from the "package maintainer. " as stated in that paragraph.

Reply:
I'll try to think of some appropriate words (although if anyone has suggestions please suggest them). For the alternatives you do get support from the package maintainer but the maintainer is different (for example the package maintainer of the NodeSource packages is NodeSource). 

Maybe keep the sentence and explicitly call out the package maintainer in the bullets?
e.g., 
> These resources provide packages compatible with CentOS, Fedora, and RHEL.
> * [Node.js snaps](#snap) maintained by https://github.com/nodejs/snap
> * [Node.js binary distributions](#debian-and-ubuntu-based-linux-distributions) maintained by [NodeSource](https://github.com/nodesource/distributions)

Reply:
@mhdawson I've updated the wording of the bullet points to explicitly state where the alternatives are maintained. PTAL.